CHANGELOG.md

```diff
@@ -5,8 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 ### Added
-- authentication/OfflineTokenRefresh.js - refresh oauth2 offline tokens
-- httpsender/AddBearerTokenHeader.js - refresh oauth2 offline tokens
+- authentication/OfflineTokenRefresh.js > refresh oauth2 offline tokens
+- httpsender/AddBearerTokenHeader.js > refresh oauth2 offline tokens
+- targeted/WordPress Username Enumeration.js > A targeted script to check for WordPress Username Enumeration via author archives
```

Comment:
This doesn't seem right.

Reply:
It was actually because we used `>` as separator.